include/z64.h

```diff
@@ -1136,6 +1136,40 @@ typedef struct {
     /* 0x01E2 */ char unk_1E2[0x06];
 } TitleContext; // size = 0x1E8
 
+typedef struct {
+    /* 0x00 */ char* name;
+    /* 0x04 */ void (*loadFunc)(struct SelectContext*, s32);
+    /* 0x08 */ s32 entranceIndex;
+} MapEntry; // size = 0xC
```

Comment:
just want to note im changing any name with map in it to scene before this gets merged

Reply:
Noted. I suggest `SceneSelectEntry` for this particular struct, so it's not confused with other "scene" structs like scene table entries (called `Scene` currently).

Reply:
agreed. ill change stuff in a min

Reply:
changes made